gui/wxpython/gcp/manager.py

```diff
@@ -885,15 +885,22 @@ def OnEnterPage(self, event=None):
                                     self.parent.grouppage.xygroup,
                                     'VREF')
 
-            f = open(vgrpfile)
+            error_message = _(
```

Comment:
It seems the same error message is used already on 2 places in this function, could you please use the error_massage in all these places?

Reply:
All right, I fixed it.